### DISCUSSION TITLE
Running Ubuntu 18.04 container inside of a ubuntu-latest runner

### DISCUSSION THREAD

**Discussion by CyrusJavan** [community/community#46674] [Actions]
### Select Topic Area

Question

### Body

I have a hard requirement to build my application on ubuntu-18.04. However, with 18.04 support quickly running out I need to come up with a solution that will allow me to use ubuntu-latest but still build my application on 18.04. I've heard it is possible to run a container within an action workflow. Has anyone done something like that? Any pointers to a blog/tutorial on how to accomplish that? 

### Update (edit)

It is actually a well supported thing to jun a workflow on a container. See https://docs.github.com/en/actions/using-jobs/running-jobs-in-a-container

**Comment by jge162** [comment]
I think it’s possible but I don’t have any resources to point you to or guides. Sorry.

**Reply by CyrusJavan** [reply]
Just updated my question I think I found the answer. See the link in question.
